src/main/java/edu/harvard/iq/dataverse/authorization/AuthenticationProvidersRegistrationServiceBean.java

```diff
@@ -307,5 +349,75 @@ public boolean isOrcidEnabled() {
         return oAuth2authenticationProviders.values().stream().anyMatch( s -> s.getId().toLowerCase().contains("orcid") );
     }
     */
+    
+    private String lookupShibbolethEntityId() {
+        
+        String baseUrl; 
+        if (FeatureFlags.SHIBBOLETH_USE_LOCALHOST.enabled()) {
+            baseUrl = "http://localhost";
```

Comment:
`http://localhost` is hard coded here and in the shibservicebean (within the string `http://localhost/Shibboleth.sso/DiscoFeed`). Should `http://localhost` be factored into a common location? Should `https://localhost` (https instead of http) be an option?